backend/src/nodes/utils/ncnn_param_schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "AbsVal": {
-    "weightOrder": []
+    "weightOrder": {}
```

Comment:
Out of curiosity, where do the changes to this schema come from?

Reply:
What do you mean, where do they come from? I just needed weightOrder to be a dict instead of a list so I could associate the weights with their permitted data types. This schema is just something we made based off the NCNN operators documentation.